engine/src/core/input/input_manager.cpp

```diff
@@ -41,9 +41,9 @@ namespace Bald::Input {
         std::memcpy(m_MouseButtonsState.begin(), m_MouseButtons.begin(), MAX_MOUSE_BUTTONS * sizeof(m_MouseButtons[0]));
     }
 
-    void InputManager::EmitKeyPressedEvent(unsigned keycode) noexcept {
+    void InputManager::EmitKeyPressedEvent(unsigned keycode, bool isRepeated) noexcept {
         if(keycode >= MAX_KEYS) CORE_LOG_WARN("[InputManager] Wrong key id");
```

Comment:
I think you could change it to assert

Reply:
I don't think that's necessary. If somebody uses some weird type input device, we don't want to halt the engine.